standalone-e2e-test/pom.xml

```diff
@@ -0,0 +1,94 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.github.lammaer</groupId>
+  <artifactId>standalone-e2e-test</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  
+  <parent>
+	 <groupId>com.github.kovaku</groupId>
```

Comment:
The indentation is not consistent in this file, maybe you could use checkstyle in your project.